### PR TITLE
Fixed up luabackend

### PR DESCRIPTION
**Pull request by tcely**
luabackend wasn't responding with the test case

### Checklist
I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled and tested this code
- [X] included documentation (including possible behaviour changes)
- [x] documented the code
- [x] added or modified regression test(s)
- [x] added or modified unit test(s)

### CODE REVIEW

File: modules/luabackend/minimal.cc [hunk 1]
```diff
@@ -188,6 +188,7 @@ bool LUABackend::getSOA(const DNSName &name, SOAData &soadata) {
     }
 
     soadata.db = this;
+    soadata.qname = name;
```
Comment (Habbie) [inline modules/luabackend/minimal.cc]: Hi! Can you PR this fix separately so we can merge it before 4.1.0? Thanks!

Comment (Habbie) [inline modules/luabackend/minimal.cc]: Nevermind, somebody already did in #5556, thanks!

**Comment by pieterlexis** [comment]
If you could rebase this to the current master, we'd like to merge it

**Comment by tcely** [comment]
@pieterlexis Rebased and passing.

**Comment by tcely** [comment]
Travis build fails for reasons unrelated to my changes:

```
Jun 04 07:04:38 Unable to bind to UDPv6 socket ::1: Cannot assign requested address
```

```
Checking bad value for record type SOA test #1
test-nameserver_cc.cc(53): error in "test_AddressIsUs6": check AddressIsUs(local3) == true failed [false != true]
Test case test_QueryCacheClean did not check any assertions
```

**Comment by aerique** [comment]
> Travis build fails for reasons unrelated to my changes:

I did a rebuild and that one succeeded.

**Comment by tcely** [comment]
TODO: search for uses of `lua-logging-query` and fix them after merging with master.

**Pull request merged**